bin/allowlist

```diff
@@ -0,0 +1,5 @@
+telemetry/core
```

Comment:
This file isn't an executable so in my opinion should not live in `bin`.

Reply:
I agree, I'll move this to the project root. However, it seems like this should belong in the root of mozilla-pipeline-schemas.